src/utils/socket_operations.py

```diff
@@ -11,20 +11,21 @@
 """
 
 import zmq
+import pickle
 
 
-def create_sockets(identities, router_addr):
+def create_sockets(router_addr, *identities):
     """
     Creates a DEALER socket for each identity in the list argument. Each socket is then connected
     to the router
 
-    :param      identities:     Unique identities to give to sockets.
-    :type       identities:     list
     :param      router_addr:    Address of the router socket
     :type       router_addr:    str
+    :param      identities:     Unique identities to give to sockets.
+    :type       identities:     tuple
 
     :returns:   Newly created and connected sockets.
-    :rtype:     list
+    :rtype:     list or str
```

Comment:
Does it return a `str`? Isn't it just a `zmq.socket` if a single identity is passed?